CMakeLists.txt

```diff
@@ -566,6 +566,9 @@ if (MFC_DOCUMENTATION)
     install(DIRECTORY   "${CMAKE_CURRENT_SOURCE_DIR}/docs/res"
             DESTINATION "docs/mfc")
 
+    install(FILES       "${CMAKE_CURRENT_SOURCE_DIR}/docs/robots.txt"
```

Comment:
Looks like I forgot this when looking at your previous PR!

Reply:
You didn't forget this. The sitemap was fine, but I pushed this change afterwards direct to master because it seemed like such a trivial change and I thought I knew what I was doing... I was wrong, but it should work now.